packages/voter-dapp/run_tests.sh

```diff
@@ -48,9 +47,9 @@ echo '#                                                                  #'
 echo '# 2/16. Submitting price requests                                  #'
 echo '#                                                                  #'
 echo -e '####################################################################\n'
-yarn run truffle exec ./scripts/local/RequestOraclePrice.js --network test --identifier USDETH-Commit+Reveal --time 1570000000
-yarn run truffle exec ./scripts/local/RequestOraclePrice.js --network test --identifier USDETH-2key-Commit+Reveal --time 1570000000
-yarn run truffle exec ./scripts/local/ProposeAdmin.js --network test
+yarn run truffle exec ../core/scripts/local/RequestOraclePrice.js --network test --identifier USDETH-Commit+Reveal --time 1570000000
```

Comment:
This directory change was necessary because we're now running these commands from the voter-dapp directory. This is a little hacky as packages shouldn't directly use each other's js files. We should remove this down the road.